modules/ROOT/pages/indexes/indexes-for-full-text-search.adoc

```diff
@@ -0,0 +1,394 @@
+
+To query a full-text index, use either the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/procedures/#procedure_db_index_fulltext_querynodes[`db.index.fulltext.queryNodes`] or the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/procedures/#procedure_db_index_fulltext_relationships[`db.index.fulltext.queryRelationships`] procedures.
+
+[NOTE]
+====
+Unlike other xref:indexes/indexes-for-search-performance.adoc[search-performance indexes], full-text indexes are not automatically used by the xref:planning-and-tuning/execution-plans.adoc[Cypher query planner].
+To access full-text indexes, they must be explicitly called with the above mentioned procedures.
+====
+
+This query uses the `db.index.fulltext.queryNodes` to query the previously created full-text index for any `name` and `team` properties in the graph containing "nils":
+
+.Query a full-text index for a node property
+[source, cypher]
+----
+CALL db.index.fulltext.queryNodes("namesAndTeams", "nils") YIELD node, score
+RETURN node.name, score
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| node.name | score
+
+| "Nils Johansson" | 0.3300700783729553
+| "Nils-Erik Karlsson" | 0.27725890278816223
+
+2+d|Rows: 2
+
+|===
+
+The `score` column represents how well the index thinks that the entry matches the given query.
+Thus, in addition to any exact matches, full-text indexes return _approximate_ matches to a given query.
+This is possible because both the property values that are indexed, and the queries to the index, are processed through the analyzer such that the index can find data entities which do not exactly match the provided `STRING`.
+
+The `score` results are always returned in _descending score order_, where the best matching result entry is put first.
+
+This query uses the `db.index.fulltext.queryRelationships` to query the previously created `communications` full-text index for any `message` containing "meeting":
+
+.Query a full-text index for a relationship property
+[source, cypher]
+----
+CALL db.index.fulltext.queryRelationships("communications", "meeting") YIELD relationship, score
+RETURN type(relationship), relationship.message, score
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="3*<m"]
+|===
+| type(relationship) | relationship.message | score
+
+| "EMAILED" | "I have booked a team meeting tomorrow." | 0.3239005506038666
+
+3+d|Rows: 1
+
+|===
+
+To only obtain exact matches, quote the `STRING` you are searching for:
+
+.Query a full-text index for exact matches
+[source, cypher]
+----
+CALL db.index.fulltext.queryNodes("namesAndTeams", '"Nils-Erik"') YIELD node, score
+RETURN node.name, score
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| node.name | score
+| "Nils-Erik Karlsson" | 0.7588480710983276
+2+d|Rows: 1
+|===
+
+Lucene also allows the use of logical operators, such as `AND` and `OR`, to search for terms:
```

Comment:
```suggestion
Query strings also support the use of logical operators (`AND`, `OR`, `NOT`):
```
Is my list exhaustive?

Reply:
I think you missed `XOR`?

Reply:
We need to check the Lucene docs. XOR on strings would be quite odd though.

Reply:
Lucene (https://lucene.apache.org/core/2_9_4/queryparsersyntax.html#Boolean%20operators) lists the following:
AND, OR, NOT, +, and -

Reply:
Great, updated my suggestion.